CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -158,22 +158,17 @@ CF_PRIVATE CFIndex __CFActiveProcessorCount();
 #endif
 
 #if DEPLOYMENT_TARGET_WINDOWS
-#define __builtin_unreachable() do { } while (0)
+#if !defined(__GNUC__)
```

Comment:
Because you could be using a GNU compatible compiler on Windows (e.g. clang).  *-windows-* other than windows-msvc will define __GNU__ even though the code is being compiled for Windows.